test-fixtures/generate-metrics.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 BUILD_LOG_FILE="$1"
 TYPE_LOG_FILE="$2"
-THRESHOLD_UNIT_TEST=16
-THRESHOLD_XCUITEST=16
+THRESHOLD_UNIT_TEST=14
+THRESHOLD_XCUITEST=14
```

Comment:
From time to time the number of warnings reach 15 for some reasons ([example](https://mozilla.slack.com/archives/CAFC45W5A/p1682615240507429)). I haven't found the reason of this sporadic BR warning increase, so I would lower this to 15 instead of 14 to make sure we don't block PRs for nothing.

Reply:
The threshold was 16 before which would cover the warnings up to 15. Now I fixed two warnings so I think we should be good with 14.